_includes/sections/working-here/your-pay-pension-and-other-benefits.md

```diff
@@ -108,38 +108,103 @@ your holiday allowance shown on your BreatheHR dashboard.
+This policy covers both short and longer term sickness absences. Our aim is to
+look after the health and wellbeing of all dxw staff and the needs of the
+company.
+
+##### Communication of sickness
+
+If you’re sick, you must let your line manager and a member of the HR team know
+by 10:00am or as soon as you reasonably can after that. This can be done via
+email, directly on Slack, phone call or text. Line managers will record and
+monitor sickness for individuals within their teams.
+
+If you are unwell and know that you are going to be off for a period of time, or
+have been signed off by your GP until a certain date, you don’t need to update
+your line manager daily. Let them know when you will be checking in again so
+they know when to expect to hear from you.
+
+When you return to work, you must check your sickness has been correctly
+recorded in BreatheHR, update it if necessary and mark yourself as returned to
+work. Your line manager will then review and close the sick leave. If you're
+sick for more than 7 calendar days in a row, you'll also need to provide a fit
+note.
+
+##### Absence meetings
+
+Your line manager should check-in with you on a regular basis about your
+wellbeing, and catch up with you after any sickness absence to see how you are.
+If you're sick for more than 7 working days over a rolling 12 month period, your
+line manager will invite you to an absence meeting to discuss your health. Where
+appropriate, we’ll talk about how we can support your return to work and any
+temporary or permanent adjustments which might help improve your wellbeing and
+support you to need fewer absences. We might also agree an action plan and / or a
+review period. Where we think it would be helpful, we’ll seek medical advice via
+an Occupational Health advisor.
+
+If you have a long term health condition and would like to discuss reasonable
+adjustments and ways of working that will help, we welcome a conversation at any
+point. The sooner we know, the sooner we can support you.
+
+##### Pay
+
+Within any rolling 12 month period, a person will receive:
+
+- full pay for the first 12 days of sickness absence
+- half pay (but no less than living wage) for the next 8 days of sickness
+  absence
+- real living wage for the next 10 days of sickness absence
+- statutory sick pay pay after 30 days of sickness absence
+
+There'll be circumstances where we'll make exceptions, because it's not possible
+to write a policy that is fair in every situation. This structure will form our
+starting point for those decisions. All decisions on exceptions will be made by
+the Directors’ group. Our aim is to ensure that we’re treating everyone fairly
+and consistently, while being aware of our financial requirements.
+
+##### Long term medical treatment
+
+Regular and open communication with your line manager is important so
+we can support you in the best possible way through a period of long term
+medical treatment (a period of more than 2 weeks).
```

Comment:
```suggestion
medical treatment (a period of more than 2 consecutive weeks).
```

Or is it 10 total days for treatment spread out over many weeks, or something else?